### PR TITLE
Allow multi word arguments to be passed to runs

### PR DESCRIPTION
**Pull request by kpamnany**

### CODE REVIEW

**Comment by simonbyrne** [comment]
What was the usecase for this?

**Comment by kpamnany** [comment]
I want to enable diagnostics for a couple of experiments run in Slurm CI to prevent all these silly little bugs that are creeping in. But enabling diagnostics requires passing `--diagnostics default` as a parameter to the run. And without this, we get `'--diagnostics default'` which breaks, as [here](https://gist.github.com/climabot/9035c613f5e2660070e95e65ae1b0f3a).

**Comment by simonbyrne** [comment]
Ah. You can do `["--diagnostics", "default"]`, but I can see that is suboptimal.

Does Argparse.jl to let you write `--diagnostics=default`?

File: slurmci.jl [hunk 1]
```diff
@@ -36,9 +36,12 @@ function load_jobs(sha::String, tag::String)
     function create_test_job(sname, entry)
         cmd = `$(sname) $(entry["file"])`
         for arg in entry["args"]
-            push!(cmd.exec, arg)
+            arg_parts = split(arg)
+            for arg_part in arg_parts
+                push!(cmd.exec, arg_part)
+            end
```
Comment (simonbyrne) [inline slurmci.jl:42]:
```suggestion
            append!(cmd.exec, Base.shell_split(arg))
```
`Base.shell_split` handles standard shell quoting

**Comment by simonbyrne** [comment]
It does look like it lets you use `--diagnostics=default`

**Comment by kpamnany** [comment]
Ah. Then this isn't really necessary then!

**Pull request closed without merging**